services/movie_session.py

```diff
@@ -0,0 +1,42 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from db.models import Movie, CinemaHall, MovieSession
+
+
+def create_movie_session(movie_show_time: datetime,
+                         movie_id: int,
+                         cinema_hall_id: int
+                         ) -> None:
+    MovieSession.objects.create(show_time=movie_show_time,
+                                cinema_hall=CinemaHall.objects.get(
+                                    id=cinema_hall_id),
+                                movie=Movie.objects.get(id=movie_id)
```

Comment:
This code is loading the database unnecessarily. According to the checklist, you should avoid loading the DB when it's not needed. Instead of using `CinemaHall.objects.get(id=cinema_hall_id)` and `Movie.objects.get(id=movie_id)`, you can directly assign the IDs to the foreign key fields.